core/git_mixins/history.py

```diff
@@ -63,12 +65,13 @@ def log(self, author=None, branch=None, file_path=None, start_end=None, cherry=N
 
         return entries
 
-    def commit_generator(self, limit = 6000):
+    def commit_generator(self, limit = 6000, follow=False):
```

Comment:
Yes. Generally, I prefer to pass several arguments then to set instance variables. On [log](https://github.com/divmain/GitSavvy/blob/53b11726f82733d186feaba0a3f0a90335b7ac27/core/git_mixins/history.py#L28-L28) it looks a little ugly, but it is very nice and explicit what's all the arguments.